packages/cascara/src/modules/DataCheckbox/DataCheckbox.js

```diff
@@ -31,26 +31,39 @@ const DataCheckbox = ({
   const { isEditing, formMethods } = useContext(ModuleContext);
   const checkbox = useCheckboxState({ state: Boolean(value) });
 
+  // do not add aria label if tag is used
+  const ariaLabel = isLabeled ? {} : { 'aria-label': label };
```

Comment:
```suggestion
  const ariaLabel = isLabeled ? { 'id': label } : { 'aria-label': label };
```
@lordpixel I _think_ in this scenario we can do this and not set the id in cases where an aria label is present, as it is not being used if there is no label tag linking to it.